providers/github/resources/github_package.go

```diff
@@ -12,3 +22,96 @@ func (g *mqlGithubPackage) id() (string, error) {
 	id := g.Id.Data
 	return "github.package/" + strconv.FormatInt(id, 10), nil
 }
+
+func (g *mqlGithubPackages) id() (string, error) {
+	return "github.packages", nil
+}
+
+func (g *mqlGithubPackages) list() ([]any, error) {
+	return newMqlGithubPackages(g.MqlRuntime, nil)
+}
+
+func (g *mqlGithubPackages) public() ([]any, error) {
+	visibility := "public"
+	return newMqlGithubPackages(g.MqlRuntime, &visibility)
+}
+
+func (g *mqlGithubPackages) private() ([]any, error) {
+	visibility := "private"
+	return newMqlGithubPackages(g.MqlRuntime, &visibility)
+}
+
+func (g *mqlGithubPackages) internal() ([]any, error) {
+	visibility := "internal"
+	return newMqlGithubPackages(g.MqlRuntime, &visibility)
+}
+
+func newMqlGithubPackages(runtime *plugin.Runtime, visibility *string) ([]any, error) {
+	conn := runtime.Connection.(*connection.GithubConnection)
+	orgId, err := conn.Organization()
+	if err != nil {
+		return nil, err
+	}
+
+	pkgTypes := []string{"npm", "maven", "rubygems", "docker", "nuget", "container"}
+	res := []any{}
+	for i := range pkgTypes {
+		listOpts := &github.PackageListOptions{
+			ListOptions: github.ListOptions{PerPage: paginationPerPage},
+			PackageType: github.String(pkgTypes[i]),
+			Visibility:  visibility,
+		}
+
+		var allPackages []*github.Package
+		for {
+			packages, resp, err := conn.Client().Organizations.ListPackages(context.Background(), orgId.Name, listOpts)
```

Comment:
I don't know how this is intended to be used.
But this line makes the output of this command a bit confusing to me:
```
cnquery run github repo mondoohq/packer-plugin-cnspec -c "github.packages"
```
I'd expected only the packages of the specified repo, but I got all across the org. Is this how it should be?

Reply:
Packages are not linked to repos, packages are always stored under account/org

Reply:
Ah, thanks for the background.